CRM/Event/Form/Registration/Register.php

```diff
@@ -678,7 +678,12 @@ public static function formatFieldsForOptionFull(&$form) {
       }
     }
 
+    // Do not freeze full options on the back-end registration forms
     $className = CRM_Utils_System::getClassName($form);
+    $overrideIsFull = FALSE;
+    if ($className == 'CRM_Event_Form_Participant' || $className == 'CRM_Event_Form_ParticipantFeeSelection') {
+      $overrideIsFull = TRUE;
+    }
```

Comment:
@JKingsnorth lets change it 1 line:
```
$overrideIsFull = (in_array($className, ['CRM_Event_Form_Participant', 'CRM_Event_Form_ParticipantFeeSelection']));
```

Reply:
@monishdeb you mean change that one line in this PR?

Reply:
oh I see - quicker way to say it

Reply:
Yes, but this is no longer required if we move this check below at L739 :)

Reply:
@monishdeb I think if ($form->isBackOffice) would work :-)

Reply:
Umm nope as ```CRM_Event_Form_ParticipantFeeSelection``` doesn't inherit that property. Unless we should add the property ```isBackOffice``` to ```CRM_Core_Form``` and let the child form decide wether they like to be in front or as a silent worker

Reply:
We can just add it to CRM_Event_Form_ParticipantFeeSelection I guess - if it IS always backoffice?